### PR TITLE
Update config.toml for language name standardization

### PR DESCRIPTION
**Pull request by a-mccarthy**
This PR implements the new localization standards set in #31657 for the language dropdown to include the localized language name and the English language name in parentheses in the language drop down. The localization sub group feels that this will help improve reader's ability to identify localized versions of the Kubernetes docs. 

There are 2 goals of this PR
1. Update the config.toml file to update the `languageName` variable for each localization to include `LanguageName (latin script language name)`. This will update the language drop down on the site to include both the language name and the latin script, English language name. For example 
    * Español will now appear as `Español (Spanish)`
    * Chinese will now appear as `中文 (Chinese)`
2. Add a variable, `languageNameLatinScript`, that holds the latin script name for each localization. 
    * For Español, languageNameLatinScript = "Español"
    * For Chinese, languageNameLatinScript = "Chinese"

Notes about this change:
* I didn't include English (English), because that seemed unnecessary
* Bahasa Indonesia is written (i believe) in the english latin script version already. So i didn't add a parenthetical language name for it, because it seemed unnecessary to have Bahasa Indonesia (Bahasa Indonesia). I will have to check with the localization team to see if there is anything to update here (i am not familiar with this language)

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -442,15 +453,17 @@ language_alternatives = ["en"]
 [languages.vi]
 title = "Kubernetes"
 description = "Giải pháp điều phối container trong môi trường production"
-languageName = "Tiếng Việt"
+languageName = "Tiếng Việt (Vietnamese)"
+languageNameLatinScript = "Vietnamese"
```
Comment (sftim) [inline config.toml:457]:
```suggestion
languageNameLatinScript = "Tiếng Việt"
```

File: config.toml [hunk 2]
```diff
@@ -378,7 +384,8 @@ language_alternatives = ["en"]
 [languages.de]
 title = "Kubernetes"
 description = "Produktionsreife Container-Orchestrierung"
-languageName ="Deutsch"
+languageName ="Deutsch (German)"
+languageNameLatinScript = "German"
```
Comment (sftim) [inline config.toml:388]:
```suggestion
languageNameLatinScript = "Deutsch"
```

**Review by sftim — COMMENTED** [review]
Would it be useful to also expose the bare language name in its native script? I can see that being useful.

**Comment by a-mccarthy** [comment]
@sftim thanks for the quick review! I've updated the languageNameLatinScript you called out, i thought i'd updated those in a previous commit, but looks like i missed some :)

**Review by divya-mohan0209 — COMMENTED** [review]
Just a quick question, is the scope of this piece of work delimited to the localisations that are live right now? Or are we including the ones that are being worked upon, as well i.e. Bengali?

**Comment by a-mccarthy** [comment]
@divya-mohan0209 thats a good point! This only updates the live localizations, but will be something new languages, like Bengali, would need to add.

**Comment by seokho-son** [comment]
/lgtm

:)

**Comment by sftim** [comment]
/lgtm

If you're happy to, @a-mccarthy, you could rebase<sup>†</sup> something like this:
```bash
git checkout main
git pull --ff-only
git checkout fixes-31657
saved_commit_id="$( git merge-base HEAD main )"
git reset --hard main
git checkout "${saved_commit_id}" -- config.toml content/en/docs/contribute/localization.md
git add config.toml content/en/docs/contribute/localization.md
git commit -m 'Update config.toml for language name standardization' -m $'Update languageNameLatinScript variable\nFix languageNameLatinScript\nUpdate localization standards examples'
git push --force-with-lease
```

That's optional; it's tidier because it avoids merging a merge commit.

† ish

**Comment by zacharysarah** [comment]
@a-mccarthy Thanks for this! ✨ 
/approve

**Pull request merged**